aries_cloudagent/protocols/out_of_band/v1_0/manager.py

```diff
@@ -172,7 +181,9 @@ async def create_invitation(
                         a_id,
                     )
                     message_attachments.append(
-                        InvitationMessage.wrap_message(cred_ex_rec.cred_offer.offer())
+                        InvitationMessage.wrap_message(
+                            cred_ex_rec.cred_offer.serialize()
```

Comment:
I see here it's using `cred_ex_rec.cred_offer` but at line 175 it's using `cred_ex_rec.credential_offer_dict`, is that correct?

Reply:
`cred_offer` in `V20CredExRecord` is same as `credential_offer_dict` in `V10CredentialExchange`. I think this is correct.